pom.xml

```diff
@@ -352,6 +352,7 @@
         <spotbugs.version>4.0.2</spotbugs.version>
         <spotbugs.omitvisitors>SerializableIdiom</spotbugs.omitvisitors>
         <stacktable-webjars.version>1.0.1</stacktable-webjars.version>
+        <spring-boot-starter-data-redis.version>2.3.0.RELEASE</spring-boot-starter-data-redis.version>
```

Comment:
No need for this - it is already defined in spring-boot parent.

Reply:
Fixed.